test_utilities.py

```diff
@@ -0,0 +1,57 @@
+import utilities
+
+
+class TestClass:
+    test_input = "The quick brown fox jumps over the lazy dog."
+    clf = utilities.classify_model()
+
+    def test_setup_nltk(self):
+        result = utilities.setup_nltk()
+        assert result
+
+    def test_parse_sentence(self):
+        triples, root = utilities.parse_sentence(self.test_input)
+        triples = list(triples)
+        assert (("jumps", "VBZ"), "nsubj", ("fox", "NN")) in triples
+        assert (("jumps", "VBZ"), "nmod", ("dog", "NN")) in triples
+        assert root == "jumps"
+
+    def test_classify_model(self):
+        from features import features_dict
```

Comment:
The name of the function (the original one) is a bit misleading, but I get what's happening here. This is actually testing the pre-processing of the data which is part of the `classify_model` function. I would probably add a test or two to check the successful training of the model but, to be honest, I've never seen anything like that before in unit tests. So, I'm fine with just this test.

Reply:
IMHO, the function can be named `create_model`.

@janakrajchadha , This test _does_ actually check the training of the model; by testing it. [Line 53](https://github.com/vishakha-lall/MapBot/pull/119/files#diff-9be4cb70786b00450e5ea7105c5de980R53) is at place predicting the class of the `test_input` with the trained model, which is a [class variable](https://github.com/vishakha-lall/MapBot/pull/119/files#diff-9be4cb70786b00450e5ea7105c5de980R6) as we need to use it in multiple functions.

If the prediction asserts to be `S` with the `test_input`, we can be sure `utilities.classify_model()` worked correctly. The code above the `assert` statement in `test_classify_model` is to create features for the `test_input` so that the model can be passed the features.

P.S.: This function uses only `predict` so that in the case further changes to the code introduces different models/algorithms it'll only check if it's trained correctly or not. I'm assuming the features passed to the model will remain same.